ivy/functional/frontends/jax/numpy/searching_sorting.py

```diff
@@ -61,6 +61,39 @@ def msort(a):
     return ivy.msort(a)
 
 
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes(
+    {
+        "0.4.13 and below": (
+            "float16",
+            "bfloat16",
+        )
+    },
+    "jax",
+)
+def argpartition(
+    a,
+    kth,
+    axis=-1,
+    kind="stable",
+    order=None,
+) -> ivy.Array:
+    if kind != "stable":
+        logging.warning(
+            "'kind' argument to argpartition is ignored; only 'stable' partitions are"
+            " supported."
+        )
+    if order is not None:
+        raise ivy.utils.exceptions.IvyError(
+            "'order' argument to argpartition is not supported."
+        )
+
+    return ivy.argpartition(a, kth, axis=axis)
+
+
+argpartition.support_native_out = True
```

Comment:
I don't think it supports an `out` argument natively, [reference](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.argpartition.html)